doc/source/configuration/magnum-capi.rst

```diff
@@ -0,0 +1,36 @@
+========================
+Magnum Cluster API Driver
+========================
+
+Prequisites for deploying the CAPI driver in magnum: 
+
+Management Cluster
+==================
+The CAPI driver relies on a management cluster to manage tenant kubernetes clusters. 
+The easiest way to get one is by deploying [this](https://github.com/stackhpc/azimuth-config/tree/feature/capi-mgmt-config) branch of azimuth-config, and look at the `capi-mgmt-example` environment. 
+
+
+Ensure that you have set `capi_cluster_apiserver_floating_ip: true`, as the management cluster will need an externally accessible IP.
+
+Kayobe Config 
+=============
+Ensure that your kayobe-config branch is up to date on stackhpc/yoga. 
+
+Copy the kubeconfig found at `kubeconfig-capi-mgmt-<your-environment>.yaml` to your kayobe environment (e.g. `<your-environment>/kolla/config/magnum/kubeconfig`. 
+
+Ensure that your magnum.conf has the following set: 
```

Comment:
Can we please note why, i.e. this is used to generate the application credentials config injected into the Kubernetes clusters, such that it is usable from within an OpenStack project, so you can't use the "internal API" end point here.